pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -278,6 +290,19 @@ def test_description(self) -> str:
     def predict_description(self) -> str:
         return "Predicting"
 
+    def _update_for_light_colab_theme(self) -> None:
+        if _detect_light_colab_theme():
```

Comment:
kaggle notebooks has color themes too 😄

Reply:
<img width="1789" alt="Screen Shot 2021-12-08 at 5 05 02 PM" src="https://user-images.githubusercontent.com/45285388/145205515-eca2d11d-105f-4405-ad6c-5cace8c459a6.png">

Yeah, we could figure it out in a follow up PR by fetching css tags.

Reply:
yeah these 3 are the only most popular ones I know: `kaggle`, `colab`, `jupyter-lab (default dark mode)`